docker-build-config/wait-for

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
```

Comment:
I think we need to include the License header here. It could be better if we push the upstream to update the license header for it.